bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -254,7 +254,11 @@ <h3 id="os-operators-headline" class="tween">
               <div>
                 {% if l10n_has_tag('mwc_2015') %}
                   <h4>{{ _('Firefox OS at home') }}</h4>
-                  <a href="{{ url('firefox.os.tv') }}">{{ _('The power of the Web is coming to Panasonic TVs later this year') }}</a>
+                  {% if l10n_has_tag('fxostv_update_201601') %}
+                    <a href="{{ url('firefox.os.tv') }}">{{ _('The power of the Web is coming to Panasonic TVs again in 2016') }}</a>
```

Comment:
Is the bug asking to use 'Firefox OS will power new Panasonic UHD TVs unveiled at CES' here? It's quite unclear to me.